chartfx-math/src/test/java/de/gsi/math/FastMathTests.java

```diff
@@ -188,7 +188,7 @@ private static float testSinLib(int n) {
         float[] sample = sample(n);
         long time = -System.nanoTime();
         for (int i = 0; i < n; i++) {
-            sample[i] = (float) Math.sin(sample[i]);
+            sample[i] = (float) java.lang.Math.sin(sample[i]);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.sin' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)